doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -90,14 +78,17 @@ Each subsequent start will use this initial calibration as the starting point.
 BICR configuration
 ==================
 
-The nRF54H20 PDK BICR configuration can be found in the board configuration directory as :file:`boards/arm/nrf54h20dk_nrf54h20/nrf54h20soc1_pdk_bicr.dtsi`.
+The nRF54H20 DK BICR configuration can be found in the board configuration directory as :file:`sdk-zephyr/boards/nordic/nrf54h20dk/nrf54h20dk_bicr.dtsi`.
 This file is used by the |NCS| build system to generate a corresponding HEX file.
-You can start from this file when editing the values of the devicetree properties inside your custom board folder (:file:`boards/arm/your_custom_board`), according to your board configuration.
+You can start from this file when editing the values of the devicetree properties inside your custom board folder (:file:`boards/nordic/your_custom_board`), according to your board configuration.
+
+.. caution::
+   A mismatch between the board and the configuration values in BICR can damage the device or set it in an unrecoverable state.
```

Comment:
Are there any specific things that will cause this that we should point out? Are for example a low frequency clock or power configurations the ones that can do so, while others will likely not put it into an unrecoverable state?
If so, maybe we can point out something users can keep an extra eye on?

Reply:
@David-Garcia-Polo @ingeryp @tjora Any inputs here?